### DISCUSSION TITLE
Real Betis TV

### DISCUSSION THREAD

**Discussion by fraudiay79** [iptv-org/community#350] [Show and tell]
### Channel Name

Real Betis TV

### Channel ID

BetisTV.es

### Website

https://en.realbetisbalompie.es/rtvbetis/live-tv/

### Stream URL

https://backend.flumotion.com/api/v1/player_site/?asset=991046&player=615

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by ARAB-IPTV** [comment]
https://betistv-live.flumotion.com/Hfx-qk--dpVSzNzEkM7Tew/1655582104/betistv/live_all/betistv/live_source/chunks.m3u8

**Comment by fraudiay79** [comment]
The m3u8 link expires/invalid after some time (token in the url)

**Comment by BellezaEmporium** [comment]
Quite easy to make a plugin out of it, let me see...

**Comment by sergio3623** [comment]
pueden pasar un enlace para ver en iptv el canal betis tv?ç

**Comment by freearhey** [comment]
https://betistv-live.flumotion.com/QXhdY7GQMbqvxWOkuPISCw/1663495641/betistv/live_all/playlist.m3u8
<img width="1432" alt="image" src="https://user-images.githubusercontent.com/7253922/190888492-07272abe-d76e-4d2e-b0cf-a99380f49b31.png">

**Comment by Alstruit** [comment]
https://backend.flumotion.com/api/v1/player_site/615/site.js?asset=991046 (json) -> sources -> file
 (generated m3u8). A scrapper can be implemented.

**Comment by sergio3623** [comment]
como puedo generar el m3u a partir de ese enlace?

**Comment by BellezaEmporium** [comment]
@Alstruit Getting on it this weekend.

**Comment by AntiPontifex** [comment]
Already on list: please close.

**Comment by freearhey** [comment]
https://backend.flumotion.com/api/v1/player_site/?asset=991046&player=615
<img width="537" alt="image" src="https://user-images.githubusercontent.com/7253922/232228616-7fa383e3-4cb7-4a23-a872-73ac9f09575f.png">

https://betistv-live.flumotion.com/QXhdY7GQMbqvxWOkuPISCw/1663495641/betistv/live_all/playlist.m3u8 (offline)
<img width="735" alt="image" src="https://user-images.githubusercontent.com/7253922/232228578-400f4e11-64cd-4fd1-ad2e-d2df46b0a9fd.png">
